server/src/database.js

```diff
@@ -0,0 +1,139 @@
+const Sequelize = require('sequelize');
+
+const createStore = () => {
+  const sequelize = new Sequelize({
+    dialect: 'sqlite',
+    storage: './database.sqlite',
+  });
+
+  sequelize.authenticate().then(() => {
+    console.log('Connection to the database has been established successfully');
+  }).catch(err => {
+    console.error(`Unable to connect to the database: ${err}`);
+  });
+
+  const Model = Sequelize.Model;
+
+  class User extends Model {
+  }
+
+  class Event extends Model {
+  }
+
+  User.init({
+        id: {
+          type: Sequelize.INTEGER,
+          primaryKey: true,
+          autoIncrement: true,
+        },
+        createdAt: Sequelize.DATE,
+        firstName: {type: Sequelize.STRING, allowNull: false},
+        lastName: Sequelize.STRING,
```

Comment:
Only requiring first name looks a bit strange. 

Also, will there be a separate user display name? e.g. nickname/id that user chooses.

Reply:
There is no separate user display name. So I thought firstName can be used as nickname if lastName is optional...

Reply:
I made lastName not to be null too. Thank you!